src/components/application_manager/src/application_impl.cc

```diff
@@ -507,13 +507,23 @@ void ApplicationImpl::StartStreaming(
     if (!video_streaming_approved()) {
       LOG4CXX_TRACE(logger_, "Video streaming not approved");
       MessageHelper::SendNaviStartStream(app_id(), application_manager_);
+      auto on_service_status_update_notification =
+          MessageHelper::CreateOnServiceStatusUpdateNotification(
```

Comment:
@mked-luxoft and where do you specify actual status?

Reply:
@LuxoftAKutsan Since `OnServiceUpdate` requires only `service_type` as a mandatory parameter, it is the only value that is set in `MessageHelper::CreateOnServiceStatusUpdateNotification`. Other parameters are set outside this function if needed.